easybuild/easyconfigs/g/gretl/gretl-2021c-foss-2020b.eb

```diff
@@ -0,0 +1,40 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Loris Bennett
+# ZEDAT, Freie Universitaet Berlin
+
+easyblock = 'ConfigureMake'
+
+name = 'gretl'
+version = '2021c'
+
+homepage = 'http://gretl.sourceforge.net'
+description = "A cross-platform software package for econometric analysis"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'openmp': True}
```

Comment:
@tardigradus Any reason you're not also enabling `usempi` here, like we're doing in the existing `gretl-2020a-foss-2019a.eb` easyconfig?

Reply:
The reason is stupidity :-/ 

I based the new PR on the file I used in the original PR for ```gretl-2020a-foss-2019a.eb```, rather than using the current version from Git. I'll submit an update request.

Reply:
@boegel are you happy with this one now?